src/main/java/uk/gov/hmcts/sptribs/notification/NotificationHelper.java

```diff
@@ -210,26 +210,27 @@ public Map<String, String> buildDocumentList(DynamicMultiSelectList documentList
     }
 
     private boolean isFaceToFaceFormat(Listing listing) {
-        return null != listing.getHearingFormat() && listing.getHearingFormat().equals(HearingFormat.FACE_TO_FACE);
+        return listing.getHearingFormat() != null && listing.getHearingFormat().equals(HearingFormat.FACE_TO_FACE);
     }
 
     private boolean isVideoFormat(Listing listing) {
-        return null != listing.getHearingFormat() && listing.getHearingFormat().equals(HearingFormat.VIDEO);
+        return listing.getHearingFormat() != null && listing.getHearingFormat().equals(HearingFormat.VIDEO);
     }
 
     private boolean isTelephoneFormat(Listing listing) {
-        return null != listing.getHearingFormat() && listing.getHearingFormat().equals(HearingFormat.TELEPHONE);
+        return listing.getHearingFormat() != null && listing.getHearingFormat().equals(HearingFormat.TELEPHONE);
     }
 
     public void addHearingPostponedTemplateVars(CicCase cicCase, Map<String, Object> templateVars) {
```

Comment:
Can add a test case for the `selectedHearingDateTime` to be null and assert the correct behaviour